scripts/vscripts/ZombieReborn.lua

```diff
@@ -3,6 +3,14 @@ require("util.timers")
 ZR_ZOMBIE_INFECT_TIME = 20
 ZR_ROUND_STARTED = false
 Convars:RegisterConvar("zr_knockback_scale", "5", "Knockback damage multiplier", 0)
+tWeaponConfigs = LoadKeyValues("scripts\\vscripts\\cfg\\weapons.cfg")
```

Comment:
I'd like if this was less generic. Maybe rename `zr_weapons.cfg`, or just create a dedicated `zr` folder? As I'm sure we're going to end up with other config files as well.